package.json

```
@@ -27,10 +27,10 @@
     "node": ">=10.0.0"
   },
   "peerDependencies": {
-    "postcss": "8.4.5"
+    "postcss": "8.4.6"
```

```suggestion
    "postcss": "^8.4.6"
```

We don’t want to block future upgrades.

---

Yes, let's close this PR.